src/style.css

```
@@ -1,3 +1,85 @@
+/* stylelint-disable block-no-empty */
+html {
+  box-sizing: border-box;
+}
+
 body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+ul,
+li {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  height: 60px;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding: 10px 0 10px 50px;
+}
+
+.nav {
+  padding: 22.5px 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  margin-bottom: 1px;
+}
+
+.nav__list :first-child {
+  margin-left: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: 500;
+  color: #000;
+  padding: 22.5px 0;
```

you can round this value
```suggestion
  padding: 23px 0;
```